keybase/platform_windows_test.go

```diff
@@ -63,3 +63,31 @@ func TestApplyAsset(t *testing.T) {
 	err = ctx.Apply(update, updater.UpdateOptions{}, tmpDir)
 	require.NoError(t, err)
 }
+
+func checkLegit64Code(productID string, log Log) bool {
+	if productID == "{65A3A964-3DC3-0100-0000-160621082245}" {
+		return true
+	}
+	return false
+}
+
+func checkLegit86Code(productID string, log Log) bool {
+	if productID == "{65A3A986-3DC3-0100-0000-160621082245}" {
```

Comment:
`return productID == "{65A3A986-3DC3-0100-0000-160621082245}"`

and these don't all need to be separate methods..
and make the long strings `"{65A3A986-3DC3-0100-0000-160621082245}"` consts?